pkg/net/libp2p/key_test.go

```diff
@@ -84,14 +86,12 @@ func TestOperatorPublicKeyToNetworkPublicKey(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if !reflect.DeepEqual(networkPublicKey.Curve.Params(), DefaultCurve.Params()) {
-		t.Errorf("network public key uses the wrong curve")
-	}
+	btcecPublicKey := (*btcec.PublicKey)(networkPublicKey)
```

Comment:
Can we explain this conversion as well?

Reply:
Done in 58c5230ded309eb47e97dd595427798f6ac0d71d.